packages/bot-web-ui/src/components/download/download.tsx

```diff
@@ -27,15 +27,14 @@ const Download = observer(({ tab }: TDownloadProps) => {
             classNameBubble='run-panel__info--bubble'
             alignment='bottom'
             message={popover_message}
-            zIndex={5}
+            zIndex='5'
         >
             <Button
                 id='download-button'
                 is_disabled={is_clear_stat_disabled}
-                className='download__button'
-                icon={
-                    <Icon icon='IcDownload' color={is_clear_stat_disabled && 'disabled'} className='download__icon' />
```

Comment:
if this className is not needed any more you can delete it from the styles too.

Reply:
Actually, it wasn't even being used at all.

Reply:
Sorry! Found it! Let me remove!

Reply:
there is also `download__icon` too

Reply:
Yeap, it was also unused. Removed now. The container we need so keeping it.